runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -8269,6 +8269,29 @@ done:;
 			/* Get MethodHandle for this operation from the VarHandles handleTable */
 			j9object_t handleTable = J9VMJAVALANGINVOKEVARHANDLE_HANDLETABLE(_currentThread, varHandle);
 			j9object_t methodHandle = J9JAVAARRAYOFOBJECT_LOAD(_currentThread, handleTable, operation);
+
+			if (NULL == methodHandle) {
+				prepareForExceptionThrow(_currentThread);
+
+				const char *exceptionClassName = "java/lang/UnsupportedOperationException";
+				J9Class *exceptionClass = internalFindClassUTF8(
+						_currentThread,
+						(U_8 *)exceptionClassName,
+						strlen(exceptionClassName),
```

Comment:
Although the compiler will likely fold it, you can use the `LITERAL_STRLEN` macro instead of `strlen` here.

Reply:
`LITERAL_STRLEN` doesn't work with `const char *`. it only works if a string is passed directly or as a macro.